packages/core-utils/src/time.js

```diff
@@ -34,11 +38,9 @@ export function getLongDateFormat(config) {
  * @returns {string} formatted text representation
  */
 export function formatDuration(seconds) {
-  const dur = moment.duration(seconds, "seconds");
-  let text = "";
-  if (dur.hours() > 0) text += `${dur.hours()} hr, `;
-  text += `${dur.minutes()} min`;
-  return text;
+  return formatDistanceStrict(0, 1000 * seconds, {
+    includeSeconds: false
+  });
```

Comment:
Roughly the same comment as https://github.com/opentripplanner/otp-ui/pull/271/files#r682846101

Reply:
Implemented in d7288ba2266a543eed305d19395ad909714e1b92. Will add tests to verify that it works the way you expect